docs/python_sdk_v2.0.md

```diff
@@ -122,7 +122,7 @@ for bbox in bboxes:
     pt2 = (bbox.xmax, bbox.ymax)
     cv.rectangle(im, pt1, pt2, (0, 255, 0))
 
-# conver it back to RGB
+# convert it back to RGB
 im = cv.cvtColor(im, cv.COLOR_RGB2BGR)
```

Comment:
can you also change `cv2` to `cv` in [this line](https://github.com/XailientPublic/docs/pull/30/files#diff-805089b443047978f569fa52c125b378b2eb039718d8db3d0a7fac9265735eb4R114)